### PR TITLE
Added release 1.12.4 blog post

### PR DESCRIPTION
**Pull request by AHeise**

### CODE REVIEW

File: _posts/2021-05-14-release-1.12.4.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+categories: news
+authors:
+- arvid:
+  name: "Arvid Heise"
+
+---
+
+The Apache Flink community released the next bugfix version of the Apache Flink 1.12 series.
+
+This release includes 73 fixes and minor improvements for Flink 1.12.3. The list below includes a detailed list of all fixes and improvements.
```
Comment (rmetzger) [inline _posts/2021-05-14-release-1.12.4.md]: 73 seems a high number. I only find 21 Jira links in this blog post

Comment (AHeise) [reply _posts/2021-05-14-release-1.12.4.md]: XD

File: _posts/2021-05-14-release-1.12.4.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title:  "Apache Flink 1.12.4 Released"
+date:   2021-05-14 00:00:00
```
Comment (rmetzger) [inline _posts/2021-05-14-release-1.12.4.md]:
```suggestion
date:   2021-05-21 00:00:00
```

**Review by rmetzger — APPROVED** [review]
Thanks for addressing my comments so quickly

**Pull request merged**